Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/AbilData.xml

```diff
@@ -6518,6 +6518,34 @@
                 <Flags index="ToSelection" value="1"/>
             </Button>
         </InfoArray>
+        <InfoArray index="Research7" Time="250" Upgrade="AP_ZergFlyerWeaponsLevel4">
+            <Resource index="Minerals" value="325"/>
+            <Resource index="Vespene" value="325"/>
+            <Button DefaultButtonFace="AP_zergflyerattack4" State="Restricted" Requirements="AP_LearnZergFlyerAttack4">
+                <Flags index="ToSelection" value="1"/>
+            </Button>
+        </InfoArray>
+        <InfoArray index="Research8" Time="285" Upgrade="AP_ZergFlyerWeaponsLevel5">
+            <Resource index="Minerals" value="400"/>
+            <Resource index="Vespene" value="400"/>
+            <Button DefaultButtonFace="AP_zergflyerattack5" State="Restricted" Requirements="AP_LearnZergFlyerAttack5">
+                <Flags index="ToSelection" value="1"/>
+            </Button>
+        </InfoArray>
+        <InfoArray index="Research9" Time="220" Upgrade="AP_ZergFlyerArmorsLevel4">
+            <Resource index="Minerals" value="375"/>
+            <Resource index="Vespene" value="375"/>
+            <Button DefaultButtonFace="AP_zergflyerarmor4" State="Restricted" Requirements="AP_LearnZergFlyerArmor4">
+                <Flags index="ToSelection" value="1"/>
+            </Button>
+        </InfoArray>
+        <InfoArray index="Research10" Time="220" Upgrade="AP_ZergFlyerArmorsLevel5">
+            <Resource index="Minerals" value="450"/>
+            <Resource index="Vespene" value="450"/>
+            <Button DefaultButtonFace="AP_zergflyerarmor5" State="Restricted" Requirements="AP_LearnZergFlyerArmor5">
+                <Flags index="ToSelection" value="1"/>
+            </Button>
+        </InfoArray>

```

Comment:
Looks like a copy-paste error / adjustment necessary on the times here. Flyer weapon research time increases to L4 250 and L5 285, but armour stays at L3 220, L4 220, L5 220.

I'm in favour of just capping all L4/L5 research times to be the same as L3 research times, the issue here is just that it's inconsistent.

Reply:
Fixed the typos